src/pages/certificates/tests/create-certificate.spec.js

```diff
@@ -75,16 +76,8 @@ describe('test certificates Emulation and Generation', () => {
       userEvent.type(input, '42');
     });
 
-    it('should not show error if there are no delay', () => {
+    it('should show error', () => {
```

Comment:
should show which error?

Reply:
Is the description bad? I need to add in the description what error should be shown?

Reply:
Yes, you sholud to define what error.

Imagine if you had many test cases in which you test errors, you would write everywhere should show error?
It will be unclear for another people

Reply:
I realized. I will fix it.

Reply:
I fixed.